src/Sarif/Schemata/sarif-schema.json

```diff
@@ -1840,6 +1882,115 @@
       "required": [ "target", "kinds" ]
     },
 
+    "request": {
+      "description": "A web request object.",
+      "type": "object",
+      "properties": {
+
+        "index": {
+          "description": "The index within the run.requests array of the request object associated with this result.",
+          "type": "integer"
+        },
+
+        "protocol": {
+          "description": "The request protocol. Example: 'http'.",
+          "type": "string"
+        },
+
+        "version": {
+          "description": "The request version. Example: '1.1'.",
+          "type": "string"
+        },
+
+        "uri": {
+          "description": "The URI to which the request is directed.",
+          "type": "string",
+          "format": "uri"
+        },
+
+        "method": {
+          "description": "The HTTP method. Well-known values are 'GET', 'PUT', 'POST', 'DELETE', 'PATCH', 'HEAD', 'OPTIONS', 'TRACE', 'CONNECT'.",
+          "type": "string"
+        },
+
+        "headers": {
+          "description": "The request headers.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+
+        "parameters": {
+          "description": "The request parameters .",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+
+        "body": {
+          "description": "The body of the request.",
+          "type": "string"
+        },
+
+        "properties": {
+          "description": "Key/value pairs that provide additional information about the request.",
+          "$ref": "#/definitions/propertyBag"
+        }
+      }
+    },
+
+    "response": {
+      "description": "A web response object.",
+      "type": "object",
+      "properties": {
+
+        "index": {
+          "description": "The index within the run.responses array of the response object associated with this result.",
+          "type": "integer"
+        },
+
+        "protocol": {
+          "description": "The response protocol. Example: 'http'.",
+          "type": "string"
+        },
+
+        "version": {
+          "description": "The response version. Example: '1.1'.",
+          "type": "string"
+        },
+
+        "statusCode": {
+          "description": "The response Status Code. Example: 404.",
```

Comment:
>Status Code [](start = 39, length = 11)

No caps. Just "The status code"

Reply:
Larry, its possible you are looking at iteration 3? i fixed it in 4 :)

---
In reply to: [273724036](https://github.com/Microsoft/sarif-sdk/pull/1419#discussion_r273724036) [](ancestors = 273724036)

Reply:
Ah, I was looking at an earlier iteration.

---
In reply to: [273724036](https://github.com/Microsoft/sarif-sdk/pull/1419#discussion_r273724036) [](ancestors = 273724036)